src/pages/profile.tsx

```diff
@@ -129,30 +115,72 @@ const Profile: NextPage = () => {
     },
     resolver: zodResolver(onboardSchema),
   });
+  const { data: user } = trpc.useQuery(["user.me"], { refetchOnMount: true });
 
-  const [suggestions, setSuggestions] = useState<Feature[]>([]);
-  const [selected, setSelected] = useState({ place_name: "" });
+  const [companyAddressSuggestions, setCompanyAddressSuggestions] = useState<
+    CarpoolFeature[]
+  >([]);
   const [startAddressSuggestions, setStartAddressSuggestions] = useState<
-    Feature[]
+    CarpoolFeature[]
   >([]);
-  const [startAddressSelected, setStartAddressSelected] = useState({
-    place_name: "",
-  });
+
+  const [companyAddressSelected, setCompanyAddressSelected] =
+    useState<CarpoolAddress>({
+      place_name: "",
+      center: [0, 0],
+    });
+  const [startAddressSelected, setStartAddressSelected] =
+    useState<CarpoolAddress>({
+      place_name: "",
+      center: [0, 0],
+    });
+
   const [companyAddress, setCompanyAddress] = useState("");
   const updateCompanyAddress = useMemo(
     () => debounce(setCompanyAddress, 1000),
     []
   );
+
   const [startingAddress, setStartingAddress] = useState("");
   const updateStartingAddress = useMemo(
     () => debounce(setStartingAddress, 1000),
     []
   );
 
+  console.log(startAddressSuggestions);
+
+  useEffect(() => {
+    if (!user) return;
+
+    setStartAddressSelected({
+      place_name: user.startAddress,
+      center: [user.startCoordLng, user.startCoordLat],
+    });
+    setCompanyAddressSelected({
+      place_name: user.companyAddress,
+      center: [user.companyCoordLng, user.companyCoordLat],
+    });
+
+    reset({
```

Comment:
what does reset do?

Reply:
Reset is pulled from our form inputs, and resets the default state of the components to include the user.me query